### PR TITLE
Night At The Orchestra [All the Instruments]

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## The Goal? Every Single Midi Instrument.
We have the technology. I have the willpower. Why not?

Harp, Violin, Viola, Cello, and Contrabass and other sprites courtesy of Hyenh.

**Screenshots**

image
image
instrumetshowoff

**Changelog**

:cl:
- add: Added a ton of instruments.

### CODE REVIEW

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
-    outerClothing: ClothingOuterVestDetective
-    id: DetectivePDA
-    ears: ClothingHeadsetSecurity
-    pocket1: ForensicPad
-    pocket2: ForensicScanner
-    belt: ClothingBeltHolster
-  innerclothingskirt: ClothingUniformJumpskirtDetective
-  satchel: ClothingBackpackSatchelSecurityFilled
-  duffelbag: ClothingBackpackDuffelSecurityFilled
+- type: job
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Roles/Jobs/Security/detective.yml:1]: mfw this says there's a diff even though nothing changed

**Comment by ike709** [comment]
\> compelling us to arrange a 120-player symphony orchestra

**Comment by EmoGarbage404** [comment]
> > compelling us to arrange a 120-player symphony orchestra

that is exactly what i am doing and if some one doesn't organize it at some point then we have failed as a remake:tm:.

**Comment by EeveeTea** [comment]
We can have applause in the form of "canned applause" which is a reference to canned laughter. It would probably work something like the bike horn that's already in. We could even palette swap it and add laughter as well for either the clown or a future comedian role.

**Comment by metalgearsloth** [comment]
@EmoGarbage404 you coming back to this?

**Comment by EmoGarbage404** [comment]
> @EmoGarbage404 you coming back to this?

Yeah, just waiting patiently for midi refactor.

**Comment by Zumorica** [comment]
> > @EmoGarbage404 you coming back to this?
> 
> Yeah, just waiting patiently for midi refactor.

Don't wait, it's gonna be a while until I get back to that

**Comment by EmoGarbage404** [comment]
this isn't every single one but dear god i'm burning out so i'll leave the rest for another time

**Pull request merged**